### ISSUE TITLE
`WENO5` is very different from other advection schemes

### ISSUE THREAD

**Issue by francispoulin** [CliMA/Oceananigans.jl#2098]
To get the string name of an advection scheme I can do something like the following:

```
scheme1 = UpwindBiasedThirdOrder()
string(scheme1)[1:end-2]
```
which produces "UpwindBiasedThirdOrder".  I am doing this to get titles in figures to compare the results of the different schemes.

However, when I try doing something similar with `WENO5`, it fails because it is designed very differenetly:

```
scheme2 = WENO5()
string(scheme2)
```

which instead produces

"WENO5 advection scheme with: \n    ├── X regular \n    ├── Y regular \n    └── Z regular"

I am not suggesting anything is wrong but can someone suggest how I might be able to get string names from these advection schemes more easily?  

@simone-silvestri ?

**Comment by glwagner** [comment]
@simone-silvestri

**Comment by simone-silvestri** [comment]
Makes sense, I'll change the print function to make it consistent with other advection schemes

**Comment by francispoulin** [comment]
Thank you @simone-silvestri !

**Comment by navidcy** [comment]
Let's close the issue when #2099 is merged. :)

**Comment by francispoulin** [comment]
Yes, very good idea.  Thanks for doing just that.

Francis
________________________________

**Comment by navidcy** [comment]
@francispoulin is this a better solution?

```julia
julia> scheme1 = UpwindBiasedThirdOrder()
UpwindBiasedThirdOrder()

julia> scheme2 = WENO5()
┌ Warning: defaulting to uniform WENO scheme with Float64 precision, use WENO5(grid = grid) if this was not intended
└ @ Oceananigans.Advection ~/Research/Oceananigans.jl/src/Advection/weno_fifth_order.jl:145
WENO5 advection scheme with:
    ├── X regular
    ├── Y regular
    └── Z regular

julia> string(typeof(scheme1).name.wrapper)
"UpwindBiasedThirdOrder"

julia> string(typeof(scheme2).name.wrapper)
"WENO5"
```

**Comment by navidcy** [comment]
perhaps we rename the issue to something along the lines of `show() method for `WENO5` is very different from other advection schemes`?

**Comment by francispoulin** [comment]
Closed after the disucssion in #2099
